src/components/layer-controller/LayerController.js

```diff
@@ -368,7 +371,10 @@ export default function LayerController(props) {
   const visibleSetting = typeof visible === 'boolean' ? visible : true;
   const Visibility = visibleSetting ? VisibilityIcon : VisibilityOffIcon;
   // Only show Volume tabs if 3D is available.
-  const useVolumeTabs = !disable3d && shape[labels.indexOf('z')] > 1;
+  const hasViewableResolutions = Boolean(Array.from({
+    length: loader.length,
```

Comment:
This is evaluating to `false` for the skinny TIFF file (will send link on slack), and the volume tabs are not showing up. `loader.length` is `undefined`. Is that expected for this file?

![Screen Shot 2021-08-31 at 3 07 24 PM](https://user-images.githubusercontent.com/7525285/131561618-565bb05e-7833-47b8-a200-1f2e543054e3.png)

Reply:
Marking as resolved (discussed on slack)